packages/transformers/html/src/dependencies.js

```diff
@@ -18,7 +18,8 @@ const ATTRS = {
     'iframe',
     'embed'
   ],
-  href: ['link', 'a', 'use'],
+  // Using href with <script> is described here: https://developer.mozilla.org/en-US/docs/Web/SVG/Element/script
+  href: ['link', 'a', 'use', 'script'],
   srcset: ['img', 'source'],
   poster: ['video'],
   'xlink:href': ['use', 'image'],
```

Comment:
Do we need to add `'script'` here as well? Looks like SVGs support both `href` and `xlink:href`.

Reply:
Also linking the same suggestion by @andersk — https://github.com/parcel-bundler/parcel/issues/3572#issuecomment-538673727

I think you guys are right. I didn't bother to check it and assumed that SVG2 is the current recommendation.

OMW fixing it!

Reply:
Done. :tada: